src/app/admin-panel/admin-panel/teachers/teachers.component.ts

```diff
@@ -1,15 +1,43 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, OnInit } from "@angular/core";
+import { TeachersService } from "../../../services/teachers.service";
+import { Iteachers } from "src/app/services/teachers";
 
 @Component({
-  selector: 'app-teachers',
-  templateUrl: './teachers.component.html',
-  styleUrls: ['./teachers.component.scss']
+  selector: "app-teachers",
```

Comment:
what the reason of this change?

Reply:
seems weird, don't remember when i did it, there is no reason